frontend/src/components/Header/presenter.jsx

```
@@ -0,0 +1,19 @@
+import {Button} from "@blueprintjs/core";
+import React from "react";
+import './style.css'
+
+export default function header(props) {
```

ESLint

[error] Missing space before function parentheses.

(space-before-function-paren)